calculator.js

```diff
@@ -0,0 +1,47 @@
+const operators = ['+', '-', '*', '/'];
```

Comment:
This variable currently isn't being used. I would suggest refactoring the code to use it. 

This could have been included in the function itself, or maybe extracted from a constants file instead of being added as a global variable.